deps/icu-small/source/common/mlbe.cpp

```diff
@@ -180,7 +180,7 @@ int32_t MlBreakEngine::initIndexList(const UnicodeString &inString, int32_t *ind
     }
     int32_t index = 0;
     int32_t length = inString.countChar32();
-    // Set all (lenght+4) items inside indexLength to -1 presuming -1 is 4 bytes of 0xff.
```

Comment:
This fix would need to be made in the upstream dependency. This change should be removed from this PR.